docs/deployment/create_bucket.md

```diff
@@ -8,9 +8,7 @@ hide_title: true
 
 Buckets can be created on the command line with [AWS CLI][cli] or via the web interface on the [AWS console][web].
 
-When creating a protected bucket (a bucket containing data which will be
-served through the distribution API), make sure to enable S3 Server Access
-Logging. See [EMS Reporting](ems_reporting.md) for more details.
+When creating a protected bucket (a bucket containing data which will be served through the distribution API), make sure to enable S3 Server Access Logging. See [EMS Reporting](ems_reporting.md) for more details.
```

Comment:
Should we link directly from "S3 Server Access Logging"? Or do we want people to go to "EMS reporting" first?

Reply:
Yeah I thought about that and may have settled on the wrong option.   My thought process was that I didn't want to explain why they had to do it here, but looking at it again. I'd be surprised to be dumped at ems reporting from that link.  I can change it.

Reply:
ff9fb8c2f